test/flint/fmpq-test.jl

```diff
@@ -1,3 +1,13 @@
+function test_elem(R::FlintRationalField)
+   n = fmpz(2)^rand(1:100)
+   return fmpz(rand(-n:n)) // fmpz(rand(fmpz(1):n))
```

Comment:
Could be the slightly simpler 
```julia
   return rand_bits(ZZ, rand(0:100))/rand_bits(ZZ, rand(1:100))
```

Reply:
Thanks, I didn't know about `rand_bits`!